### PR TITLE
xml_variable_list into xml_variable

### PR DESCRIPTION
**Pull request by Setnea**
Adding VariableList class #71

### CODE REVIEW

File: python/modules/xml_parser.py [hunk 1]
```diff
@@ -35,18 +35,49 @@ def get_reference(self, env): # function name requires changes
...
+        settings[key] = variable
+
+def _create_variable(param):
     key = param.get('key')
     value = param.get('value')
-    try:
-        p_type = param.get('type')
-    except:
-        p_type = "string"
+    p_type = param.get('type')
```
Comment (stork119) [inline python/modules/xml_parser.py:71]: Why did you remove try catch block ?

Comment (Setnea) [reply python/modules/xml_parser.py:71]: Because I noticed that this 'try' will always execute, in case of no "type" instance in param it will just return "None".

**Comment by Setnea** [comment]
Can I merge this branch and continue work?

**Review by stork119 — APPROVED** [review]
